packages/prompts/src/index.ts

```diff
@@ -468,14 +470,15 @@ export const groupMultiselect = <Value>(opts: GroupMultiSelectOptions<Value>) =>
 		} else if (state === 'submitted') {
 			return `${color.dim(label)}`;
 		}
-		return `${color.dim(prefix)}${color.dim(S_CHECKBOX_INACTIVE)} ${color.dim(label)}`;
+		return `${color.dim(prefix)}${isItem || selectableGroups ? color.dim(S_CHECKBOX_INACTIVE) : ""} ${color.dim(label)}`;
```

Comment:
it looks like this codebase probably uses single quotes rather than double quotes
```suggestion
		return `${color.dim(prefix)}${isItem || selectableGroups ? color.dim(S_CHECKBOX_INACTIVE) : ''} ${color.dim(label)}`;
```

Reply:
yeah: https://github.com/bombshell-dev/clack/blob/45ee73bf33b25f9a8c7e1bb6117ccc165478bf4d/.prettierrc.cjs#L4

you probably need to run `pnpm format`